CourseApp.Tests/CalculatorTest.cs

```diff
@@ -0,0 +1,111 @@
+namespace CourseApp.Tests
+{
+    using Xunit;
+
+    public class CalculatorTest
+    {
+        [Fact]
+        public void SumInt()
```

Comment:
Please use InlineData to check multiple cases

https://habr.com/ru/post/357648/